### ISSUE TITLE
adduser doesn't work while firejail in use

### ISSUE THREAD

**Issue by theotherandre** [netblue30/firejail#270]
I used the released package  0.9.36 as well as a self compiled  0.9.36 from Debian on my Jessie box. 
- Start e. g. firejail firefox (or any other program) as normal user
- su to root
- adduser test
- The result is:
- `Adding user "test" ...`
- `Adding new group "test" (1004) ...`
- `groupadd: failure while writing changes to /etc/group`
- `adduser: "/usr/sbin/groupadd -g 1004 test" returned error code 10. Exiting.`
The same with firejail --noprofile firefox
Normal writing to /etc/group works. I think, there should be no relation between a jail started by a normal user and the denial of system task outside of the jail.

**Comment by manevich** [comment]
Most likely, this is same issue as I reported: on older kernels firejail locks important system files by mounting tmpfs over them.
See issue https://github.com/netblue30/firejail/issues/261
No solution except upgrading to newer kernel.
With linux-image-4.2.0-0.bpo.1-amd64 everything fine.

**Comment by theotherandre** [comment]
Yes I agree it appears to be the same problem. I tested it with a stock kernel 3.18.26 running on my server and the problem did not occur there.
 

      Von: manevich <notifications@github.com>
 An: netblue30/firejail <firejail@noreply.github.com> 
CC: theotherandre <nowakewitz@yahoo.de>
 Gesendet: 18:21 Freitag, 29.Januar 2016
 Betreff: Re: [firejail] adduser doesn't work while firejail in use (#270)
   
Most likely, this is same issue as I reported: on older kernels firejail locks important system files by mounting tmpfs over them.
See issue #261
No solution except upgrading to newer kernel.
With linux-image-4.2.0-0.bpo.1-amd64 everything fine.—
